.github/workflows/build_and_upload_conda_packages.yaml

```diff
@@ -10,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9"]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
```

Comment:
I think we should add **Python 3.8** here as well, since `pyatlan` supports [python>=3.8.](https://github.com/atlanhq/atlan-python/blob/main/setup.py#L71)

Reply:
Will do.

Reply:
done.